requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
-pytest==2.7.2
+pytest==2.9.2
 pytest-cov==2.1.0
+git+git://github.com/tilery/utilery.git
```

Comment:
Ah ok. It seems test/test_utils.py is still referencing utilery (test_utils is currently as it was whilst still part of the original utilery package).